packages/ui-kit/src/components/PieChart/PieChart.module.scss

```diff
@@ -0,0 +1,36 @@
+@use '../../themes/typography';
+
+.rootPieChart {
+    @extend .typographyTiny;
+    overflow: auto;
+    padding: 0;
+
+    canvas {
+        height: var(--propel-component-height);
+    }
+}
+
+.pieChartTotalValue{
+    text-align: center;
+    margin-top: 10px;
+
+    span{ margin: 0; line-height: 22px; color:var(--propel-text-secondary)}
+    span:first-child{ font-size: 12px; padding-right: 5px; }
+    span:last-child{ font-size: 14px; color: var(--propel-text-primary)}
+}
+
+:export{
```

Comment:
As I mentioned in Slack, we need to reconsider the location of the Color Blue (Brand) css vars. We have a huge update from Robby with [Foundations](https://www.figma.com/file/1xgtbl9L9udIHfjg2FZOLv/UI-Style-Guide?type=design&node-id=5-14764&mode=design&t=WUhOApB6QSPB02dN-0) for our Design System. Those Foundations contain the common Design Tokens which we should export somehow to UI Kit. Previously, we did it manually. It’s not very convenient, especially when the list gets bigger. I’m currently working on an automated solution (like graphql-codegen but for Figma and css). It makes sense to store these variables in some single place like [packages/ui-kit/src/themes/_common.scss](https://github.com/propeldata/ui-kit/blob/main/packages/ui-kit/src/themes/_common.scss). Also, in order to use type autocompletion, we need to [define types for the tokens](https://github.com/propeldata/ui-kit/blob/main/packages/ui-kit/src/themes/themeTokens.ts). I’m going to eliminate this boring routine via this new automated solution. I expect an intermediate solution today. Let's sync again when I get it.